rosbridge_server/scripts/rosbridge_websocket_client.py

```diff
@@ -64,46 +61,56 @@ def wsconnection_cb(self, conn):
         self.conn = conn.result()
         # TODO check result
         self.conn.on_message = self.message
-        self.send_message('{"op":"proxy"}')
+        self.conn.write_message('{"op":"proxy","user_auth":"True"}')
```

Comment:
@pablisho shouldn't the value of 'user_auth' here depend on the value of the user_auth global? it seems hardcoded